platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/VisualStudioProject.java

```diff
@@ -55,23 +53,19 @@ public interface VisualStudioProject extends Named, Buildable {
     /**
      * Configuration for the generated project file.
      */
-    @Internal
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a `@Nested` bean.